projects/smoke_drv120/src/main.c

```diff
@@ -0,0 +1,57 @@
+#include <stdbool.h>
+
+#include "delay.h"
+#include "drv120_relay.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+// Toggle the following to select which data is logged
```

Comment:
The defines here aren't "selecting which data is logged", they're configuring the enable and status pins of the DRV120 IC, so your comment should say something about that instead.